siriuspy/siriuspy/diagbeam/bpm/csdev.py

```diff
@@ -61,9 +61,9 @@ class Const(_csdev.Const):
     TrigIntern = _csdev.Const.register('TrigIntern', _et.TRIGINTERN)
     LogTrigIntern = _csdev.Const.register('LogTrigIntern', _et.LOGTRIGINTERN)
     SwModes = _csdev.Const.register('SwModes', _et.SWMODES)
-    SwTagEnbl = _csdev.Const.register('SwTagEnbl', _et.SWTAGENBL)
-    SwDataMaskEnbl = _csdev.Const.register(
-        'SwDataMaskEnbl', _et.SWDATAMASKENBL)
+    SwPhaseSyncEnbl = _csdev.Const.register('SwPhaseSyncEnbl', _et.SWTAGENBL)
+    FOFBDataMaskEnbl = _csdev.Const.register(
+        'FOFBDataMaskEnbl', _et.FOFBDATAMASKENBL)
```

Comment:
We are also missing FOFBPhaseSyncEnbl, no?

Reply:
idem here